### PR TITLE
[Feature] Text component added with descriptions and examples

### PR DESCRIPTION
**Pull request by ketsappi**
- Added page for Text component 
- Transifex source file and translated Finnish added
- English (dummy) translations file pulled from the Transifex and added to the project

### CODE REVIEW

File: locale/fi/text.json [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    {
+      "title": "Kirjasintyyppi (Font)",
+      "paragraphs": [
+        {
+          "image.key": "",
+          "image.alt": "",
+          "text": "Suomi.fi-tyyleissä sekä käyttöliittymäkomponenteissa käytetty kirjasin Source Sans Pro on Googlen välittämä open source fontti, kaikki sen leikkaukset ovat ilmaisia ja vapaasti käytettävissäsi (Open Font License). Kirjasimen voi ladata omalle työasemalle Google Fontsista."
+        },
+        {
+          "image.key": "",
```
Comment (aappoalander) [inline locale/fi/text.json]: I really dislike the habit of leaving empty structures in translation files. In my opinion it's a lot of hassle trying to figure out what goes where when there are a lot of empty placeholders. And not such a big deal to add new structures when content changes.

The only thing that favours this kind of style is the possibility for non technical people to add text, but in any case, a release is required which involves the development workflow. 

I think we should think about this in a bit more detail and try to come up with a better workflow.

File: src/pages/components/text.tsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+                style={{
+                  display: 'flex',
+                  flexDirection: 'column',
+                  flex: 1,
+                }}
+              >
+                <Text>Body text</Text>
+                <Text.bold>Body Text Bold</Text.bold>
+                <Text smallScreen>Body Text Small</Text>
+                <Text.bold smallScreen>Body Text Small Bold</Text.bold>
```
Comment (aappoalander) [inline src/pages/components/text.tsx]: I assume these styles are coming from ui-components library? Some design tokens were already renamed, but I guess the  changes are not yet available at this end.

File: src/pages/components/text.tsx [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+          <ComponentDescription
+            mainTitle={t('lead.title')}
+            description={t('lead.description')}
+            exampleFirst={false}
+            filterProps={[]}
+          >
+            <ComponentExample>
+              <div
+                style={{
```
Comment (aappoalander) [inline src/pages/components/text.tsx]: Is this something that repeats for every component? While not wrong in any way, I'd like to watch out for things we end up duplicating in our code, even if the duplication happens in very small scale for the time being.

I think we might be gaining technical debt in this repo and should start planning improvements and refactoring.

File: locale/fi/text.json (hunk 1 above)
Comment (ketsappi) [reply locale/fi/text.json]: True, I have just continued the convention that was to be used when creating translation files.

I would also want to see much more simpler structure in translation files. Also to use unique key-names for translation files if ever possible, much more easier to find the correct  place.

File: src/pages/components/text.tsx (hunk 2 above)
Comment (ketsappi) [reply src/pages/components/text.tsx]: Yes, this feature was branched from the latest version of the develop branch and it seems to use version **0.2.5** of the `suomifi-ui-components`.

File: locale/fi/text.json (hunk 1 above)
Comment (LJKaski) [reply locale/fi/text.json]: I'm down with coming up with a better solution for this. It's not a complete mess as it is, but could be cleaner.

File: src/pages/components/text.tsx (hunk 3 above)
Comment (LJKaski) [reply src/pages/components/text.tsx]: What exactly are you referring to?

Comment (ketsappi) [reply src/pages/components/text.tsx]: I now committed change where the is new small helper function (_ExampleBlock_) to prevent duplication.

Comment (aappoalander) [reply src/pages/components/text.tsx]: @LJKaski I was referring to small things here and there. Custom components maintained over the ones we have available in ui-components, extensive use of inline styles in components (why not use styled components?), inconsistent and mixed use of design tokens and fixed css styles and so on. I guess it's more of a gut feeling for not having a clear plan for improvements and having quite a few things not exactly "finished" or done quick and dirty => maintenance will become cumbersome at some point.

File: locale/fi/text.json (hunk 1 above)
Comment (aappoalander) [reply locale/fi/text.json]: Let's create a story for investigating this and add it to the improvements H1 epic?

File: src/pages/components/text.tsx [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+    <ComponentExample>
+      <div
+        style={{
+          display: 'flex',
+          flexDirection: 'column',
+          flex: 1,
+        }}
+      >
+        {children}
+      </div>
```
[the remark this answers is not in the document]
Comment (ketsappi) [reply src/pages/components/text.tsx]: I pushed the commit that removes the unnecessary `div`-element and uses the `styles`-prop of the `ComponentExample` instead.

**Pull request merged**